src/vector/v.to.rast.multi/v.to.rast.multi.html

```diff
@@ -0,0 +1,47 @@
+<h2>DESCRIPTION</h2>
+<em>v.to.rast.multi</em> creates a raster map for each selected numeric column
+of the attribute table of a vector map. The purpose is to provide a simple and
+efficient way to rasterize different attributes of vector maps with multiple
+numeric value columns.
+<p>
+It is a simple wrapper around <em>v.to.rast</em> and most of the options from
+that module are available. <em>v.to.rast.multi</em> works as follows: First,
+(only) the key column is rasterized with <em>v.to.rast</em>. Then reclassification
+rules are generated for all selected attribute columns and the <b>key_column</b>
+and reclass maps are created with <em>r.reclass</em>.
+<p>
+Because <em>r.relass</em> only handles integer data and rounds data with
+floating point precision, the number of significant digits to preserve during
+reclassification can be defined with the <b>ndigits</b> option. Also text labels
+can be provided in the <b>label_columns</b> option.
+<p>
+If the <b>label_columns</b> or <b>ndigits</b> are given, the number and order of
+provided values has to correspond to the number and order of the selected
+<b>attribute_columns</b>.
+
+<h2>EXAMPLE</h2>
+
+This example is based on the North Carolina sample location (nc_spm_08_grass7) which
+can be downloaded from the
+<a href="https://grass.osgeo.org/download/data/">GRASS GIS website</a>.
+
+<div class="code"><pre>
+# Create raster maps for RINGS_OK amd TRACT attribute of the census_wake2000 map
+v.to.rast.multi --o --v input=census_wake2000 type=area \
+output=vrastmulti attribute_columns="RINGS_OK,TRACT" \
+label_columns="ID,TRACTID" memory=3000 ndigits=",4" separator=","
```

Comment:
```suggestion
label_columns="ID,TRACTID" memory=3000 separator=","
```

Reply:
This should work now. Unittests pass.

Reply:
I get the exact same error as reported before. This conversation was resolved but which change fixed the error? I only see formatting and black commits after the error report, maybe you forgot? 
Also, I find it odd to express ndigits with a comma first. The definition says only integer, and only valid for floating point columns, which as reported earlier, RINGS_OK and TRACT are not. The example using ndigits won't work then.